src/content/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure.mdx

```diff
@@ -1354,6 +1355,88 @@ To pass child chart options through the command line, you need to prefix them wi
 
 </TabsPageItem>
 
+<TabsPageItem id="5">
+
+There are three ways to install the Kubernetes integration in your GKE Autopilot cluster.     
+
+<CollapserGroup>
```

Comment:
@bradleycamacho and @svetlanabrennan: What differentiates these collapsers from the guided install and Manual Helm steps from the other two tabs? I see that these aren't sequential, but maybe something that clarifies whether they're the same steps or separate